code/modules/vending/vending.dm

```diff
@@ -45,7 +58,8 @@
 	mats = 20
 	layer = OBJ_LAYER - 0.1 // so items get spawned at 3, don't @ me
 	deconstruct_flags = DECON_SCREWDRIVER | DECON_WRENCH | DECON_MULTITOOL
-	object_flags = CAN_REPROGRAM_ACCESS | NO_GHOSTCRITTER
+	object_flags = CAN_REPROGRAM_ACCESS
```

Comment:
why ghostcritter?

Reply:
I would like @Skeletonman0 's reason on this, not sure why this is the case

Reply:
i dont remember changing that, probably should have the NO_GHOSTCRITTER again

Reply:
Alright, I'll change it back.